snapcraft.yaml

```diff
@@ -1544,21 +1554,24 @@ parts:
       set -eux
       craftctl default
       cd $CRAFT_STAGE/usr
-      find . -type f,l -exec rm -f $CRAFT_PART_INSTALL/usr/{} \;
-      find . -type f,l -name "*.so*" -exec bash -c "rm -f $CRAFT_PART_INSTALL/usr/{}*" \;
+      find . -type f,l -exec rm -fv $CRAFT_PART_INSTALL/usr/{} \;
+      find . -type f,l -name "*.so.*" -exec sh -c "rm -fv $CRAFT_PART_INSTALL/usr/{}*" \;
       cd $CRAFT_STAGE/usr/lib
```

Comment:
Given that the Find command is recursive, why are we repeating it for `$CRAFT_STAGE/usr/lib` and `$CRAFT_STAGE/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR`?

Reply:
No idea, it was that way when I arrived :-D

Now seriously, this is already in `gnome-3-38-2004-sdk`, so it seems something that "slipped". Probably whoever added it just said "libraries are here and here, so I'll put a command for each folder", without realizing that `find` is recursive.